crates/lib/Cargo.toml

```
@@ -0,0 +1,76 @@
+[package]
+name = "binstall"
+description = "Library backend for cargo-binstall"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/binstall"
+version = "1.0.0"
```

Same here, I think having `1.0.0` means we cannot break the API without releasing a new major release `2.0.0`

```suggestion
version = "0.1.0"
```

---

Honestly, I really don't think release `binstall` under `1.0.0` is a good idea.